src/main/scala/mesosphere/marathon/core/volume/Volumes.scala

```diff
@@ -0,0 +1,97 @@
+package mesosphere.marathon.core.volume
```

Comment:
Sorry for being unclear. Please see the Conventions section here: https://mesosphere.github.io/marathon/docs/core-architecture.html